src/Drivers/OpenAI/Client.php

```diff
@@ -15,10 +15,12 @@ class Client
     public function __construct(
         public readonly string $apiKey,
         public readonly string $url,
+        public readonly ?string $organization = null,
     ) {
-        $this->client = Http::withHeaders([
-            'Authorization' => sprintf('Bearer %s', $this->apiKey),
-        ])->baseUrl($this->url);
+        $this->client = Http::withHeaders(array_filter([
+            'Authorization' => $this->apiKey !== '' && $this->apiKey !== '0' ? sprintf('Bearer %s', $this->apiKey) : null,
+            'OpenAI-Organization' => $this->organization,
```

Comment:
@sixlive, I think this would then have a blank value to `OpenAI-Organization` if one is not present, right? This should have the conditional re-added.

Reply:
Since we are wrapping it in `array_filter` if there isn't an organization preset and it defaults to a `null` value, the header is removed if there isn't an organization configured.

Reply:
doh => pre-coffee code reading. Yes. 🤦🏼